### PR TITLE
Adding info in Catalog doc

### PR DESCRIPTION
**Pull request by dnahirna**
## Purpose of this pull request

This pull request (PR) describes adding info in Catalog doc

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [x] Other feature set, please specify: Catalog

## Affected documentation pages

https://docs.magento.com/user-guide/configuration/catalog/catalog.html

## Links to Magento source code or PRs

https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/themes/product-video.html

### CODE REVIEW

**Comment by leslietilling** [comment]
@dnahirna There are conflicts to resolve, looks like there could be file changes already merged from earlier PR. Plus the same files for https://github.com/magento/merchdocs/pull/924 (not yet merged).

**Comment by dnahirna** [comment]
> @dnahirna There are conflicts to resolve, looks like there could be file changes already merged from earlier PR. Plus the same files for #924 (not yet merged).

Done

File: src/configuration/catalog/catalog.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Cata
 
 |Field|[Scope]({% link configuration/scope.md %})|Description|
 |--- |--- |--- |
+|Enabled|Store View|Enables Product Reviews. Options: Yes / No|
```
Comment (leslietilling) [inline src/configuration/catalog/catalog.md:26]:
```suggestion
|Enabled|Store View|Enables product reviews. Options: Yes / No|
```

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
running tests

**Pull request merged**